commit eca34b4d7155a1564ab9b6230473dc26241a7d8f
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:21:53 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/lamarck/patch-machistawrap.diff

diff --git a/src/machista1.0/machista_wrap.c b/src/machista1.0/machista_wrap.c
index 4486fe05b..4ae6f0c51 100644
--- a/src/machista1.0/machista_wrap.c
+++ b/src/machista1.0/machista_wrap.c
@@ -1971,9 +1971,11 @@ _wrap_macho_loadcmd_mlt_install_name_get(ClientData clientData SWIGUNUSED, Tcl_I
     SWIG_exception_fail(SWIG_ArgError(res1), "in method '" "macho_loadcmd_mlt_install_name_get" "', argument " "1"" of type '" "struct macho_loadcmd *""'"); 
   }
   arg1 = (struct macho_loadcmd *)(argp1);
-  result = (char *) ((arg1)->mlt_install_name);
-  Tcl_SetObjResult(interp,SWIG_FromCharPtr((const char *)result));
-  return TCL_OK;
+  if (arg1) {
+    result = (char *) ((arg1)->mlt_install_name);
+    Tcl_SetObjResult(interp,SWIG_FromCharPtr((const char *)result));
+    return TCL_OK;
+  }
 fail:
   return TCL_ERROR;
 }
